KinMigrationModule/KinMigrationModule/KinMigrationManager.swift

```diff
@@ -409,3 +429,11 @@ extension KinMigrationManager {
         kinSDKClient.deleteKeystore()
     }
 }
+
+// MARK: - Should migrate -
+
+extension KinMigrationManager {
+    fileprivate func shouldMigrate() -> Promise<Bool> {
+        return delegate?.kinMigrationManager(shouldMigrate: self) ?? Promise<Bool>().signal(true)
```

Comment:
@alongenosar you can make the code shorter by doing `Promise(true)` instead of `Promise<Bool>().signal(true)`.

Reply:
done